tests/stub/routing/test_routing_v3.py

```diff
@@ -79,3 +79,6 @@ def test_should_fail_on_empty_routing_response(self):
                 self.assertIn("routing", e.msg)
 
         self.assertTrue(failed)
+
+    def test_should_retry_write_until_success_with_leader_change_on_run_using_tx_function(self):  # noqa: E501
+        super().test_should_retry_write_until_success_with_leader_change_on_run_using_tx_function()  # noqa: E501
```

Comment:
I won't reject this change but it seems odd to only have one tests explicitly mentioned like this.